src/components/15Years/15YearsHowKivaWorks.vue

```diff
@@ -83,16 +83,11 @@ export default {
 /* stylelint-disable no-descending-specificity */
 
 .works {
-	background: #fff;
-	color: $twilight;
-
 	/* hides the horizontal scrollbar created by hacking the carousel below */
 	width: 100%;
 	overflow-x: hidden;
```

Comment:
Seems like these are prob non operational now that there isn't a "root" instance of the "works" class... prob doesn't matter too much though for this component.